cmake/cmake_options.yml

```diff
@@ -40,6 +40,8 @@ USE_MXNET_LIB_NAMING: "ON" # Use MXNet library naming conventions.
 USE_GPROF: "OFF" # Compile with gprof (profiling) flag
 USE_CXX14_IF_AVAILABLE: "OFF" # Build with C++14 if the compiler supports it
 USE_VTUNE: "OFF" # Enable use of Intel Amplifier XE (VTune)) # one could set VTUNE_ROOT for search path
+USE_BLAS: "openblas" # Build with BLAS version "openblas" # choose from : mkl, blas, atlas (linux default), openblas, apple (osx default)
```

Comment:
are you sure it's USE_BLAS='openblas'? I have tried USE_BLAS='open' and that works for me, can you double check?

Reply:
https://github.com/apache/incubator-mxnet/blob/24cce9e3c99e499b696b779cbb3b863145f473f1/make/config.mk#L120

Referred the make config file. #15691 just made it to openblas